AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/RemoteSyncEngine+IncomingEventReconciliationQueueEvent.swift

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+import Combine
+import Foundation
+
+@available(iOS 13.0, *)
+extension RemoteSyncEngine {
+    @available(iOS 13.0, *)
+    func onReceiveCompletion(receiveCompletion: Subscribers.Completion<DataStoreError>) {
+        if case .failure(let error) = receiveCompletion {
```

Comment:
prefer using `switch` rather than serial `if` to model mutually exclusive choices like picking between enum cases.

Reply:
This was changed over to a switch statement:
https://github.com/aws-amplify/amplify-ios/pull/324/files